### DISCUSSION TITLE
Does open-brush support HoloLens 2?

### DISCUSSION THREAD

**Discussion by Hanning-Liu** [icosa-gallery/open-brush#654] [Q&A] [answered]
Does open-brush support HoloLens 2?

**Comment by mikeskydev** [comment, marked as answer]
It doesn't at the moment, mainly because we've never got our hands on one! It would be great if someone who had one could contribute support.

**Reply by Hanning-Liu** [reply]
Thank you for your reply! I just opened the Unity project, I don't even know how to start. It seems like tons of work!😳
